AndroidXCI/cli/src/main/kotlin/dev/androidx/ci/cli/Main.kt

```diff
@@ -154,7 +156,9 @@ private class Cli : CliktCommand() {
             are run for a TestMatrix.
         """.trimIndent(),
         envvar = "ANDROIDX_IGNORE_EMPTY_TEST_MATRICES"
-    )
+    ).convert {
+        it.takeIf { it.isNotEmpty() }?.toBoolean() ?: true
```

Comment:
```suggestion
        if (it. isNullOrBlank()) true
        else it.toBoolean()
```

btw, if we don't call default, I think it's just `null` anyway, which we already handle?

https://ajalt.github.io/clikt/options/#default-values

Reply:
🙈

Reply:
i tried this locally but if you set env variable (which we do in action.yml), it becomes empty string rather than null.

hence i added this to check for empty string.